Source/Scene/Axis.js

```diff
@@ -0,0 +1,40 @@
+/*global define*/
+define([
+        '../Core/freezeObject'
+], function(
+        freezeObject) {
+    'use strict';
+
+    /**
+     * An enum describing the x, y, z axes.
+     *
+     * @exports Axis
+     */
+    var Axis = {
+        /**
+         * Denotes the x-axis.
+         *
+         * @type {String}
+         * @constant
+         */
+        X : 'X',
```

Comment:
Are we sure we should use strings here?  That is fine for the format, but generally not something we would do for the runtime.

Perhaps make these 0, 1, 2, and provide a static creation function?  Perhaps even move `yUpToZUp` and `zUpToZUp` here as frozen properties for better cohesion.